bedrock/firefox/templates/firefox/australis/fx38_0_5/firstrun.html

```diff
@@ -0,0 +1,137 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% add_lang_files "firefox/whatsnew_38" %}
+
+{% extends "firefox/base.html" %}
+
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
+{% block page_og_title %}
+  {{_('Choose the independent browser')}}
+{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}
+  {{_('Firefox is non-profit, non-corporate, non-compromised. Choosing Firefox isn’t just choosing a browser. It’s a vote for personal freedom.')}}
+{% endblock %}
+
+{#- Override <meta property="og:url"> for social share -#}
+{% block page_og_url %}{{ url('firefox') }}{% endblock %}
+
+{% block body_class %}mozID{% endblock %}
+
+{% block tabzilla_css %}{% endblock %}
+{% block tabzilla_js %}{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'firefox_fx38_0_5_firstrun' %}
+{% endblock %}
+
+{% block site_header %}
+  <header id="masthead">
+    {% block tabzilla_tab %}
+    <div id="tabzilla">
+      <a href="{{ url('mozorg.home') }}">Mozilla</a>
+    </div>
+    {% endblock %}
+    {% block site_header_nav %}{% endblock %}
+    {% block site_header_logo %}{% endblock %}
+    {% block breadcrumbs %}{% endblock %}
+  </header>
+{% endblock %}
+
+{% block content %}
+<section id="intro" data-funnelcake="{{ funnelcake_id }}">
+  <div class="container">
+    <div id="logo">
+      {{ high_res_img('img/firefox/australis/fx38.0.5/firefox-logo.png', {'alt': 'Firefox', 'width': '142', 'height': '147'}) }}
+    </div>
+
+    <div class="leftcol">
+      <h1>
+        {{ _('Almost done!') }}
+      </h1>
+
+      <h2>
+        {# L10n: Line breaks below are for visual formatting only. #}
+        <span class="ctaMenu">{{ _("Set up Sync and <br>you’re good to go.") }}</span>
+        <span class="ctaDirect">{{ _('Sign in to Firefox and <br>you’re good to go.') }}</span>
+      </h2>
+
+      <p>
+        {{ _('With your Firefox Account you can sync your bookmarks, passwords, open tabs and more, and access them everywhere you use Firefox.') }}
+      </p>
+    </div>
+
+    <div class="rightcol">
+      {{ high_res_img('img/firefox/australis/fx38.0.5/sync.png', {'alt': 'Firefox Sync', 'width': '219', 'height': '118'}) }}
+      <button class="button-flat-green" id="cta-signup">
```

Comment:
If JS is disabled, or if `browser.uitour.enabled` is set to false, clicking the button doesn't do anything currently. Worth changing to a link so it falls back to `/firefox/sync/`, or something similar?